src/libs/actions/Report.js

```diff
@@ -327,8 +327,24 @@ function openReport(reportID, participantList = [], newReportObject = {}) {
         }],
     };
 
+    const params = {
+        reportID,
+        emailList: participantList ? participantList.join(',') : '',
+    };
+
     // If we are creating a new report, we need to add the optimistic report data and a report action
     if (!_.isEmpty(newReportObject)) {
+        // Change the method to set for new reports because it doesn't exist yet, is faster,
+        // and we need the data to be available when we navigate to the chat page
+        onyxData.optimisticData[0].onyxMethod = CONST.ONYX.METHOD.SET;
+
+        const optimisticReportAction = ReportUtils.buildOptimisticCreatedReportAction(newReportObject.ownerEmail);
```

Comment:
NAB but this is a misnomer, because it's actually not just the reportAction but the weird 0-indexed object. We'll clean this up soon as part of https://github.com/Expensify/App/pull/13118, but it would be good to call this `optimisticCreatedActionData` or something else in the interim.